playbooks/ansible/roles/ctdb.setup/tasks/gpfs/main.yml

```diff
@@ -6,4 +6,3 @@
   vars:
     name: "ctdb"
     path: "{{ ctdb_lock_path }}"
-    mode: 0777
```

Comment:
Here also whatever is the default mode should suffice for CTDB cluster lock operations, right?

Reply:
Yes. Write access to the owner (root) is enough for CTDB.